compiler/rustc_target/src/spec/armv5te_unknown_linux_uclibcgnueabi.rs

```diff
@@ -0,0 +1,33 @@
+use crate::spec::{LinkArgs, LinkerFlavor, Target, TargetOptions};
+
+pub fn target() -> Target {
+    let mut post_link_args = LinkArgs::new();
+    post_link_args.insert(
+        LinkerFlavor::Gcc,
+        vec![
+            // The linker is missing the 'dlsym' symbol when we don't
+            // explicitly add '-ldl'.
+            "-ldl".to_string(),
+            "-lgcc_s".to_string(),
```

Comment:
Linking to these libraries is job of libc crate (https://github.com/rust-lang/libc/blob/master/src/unix/mod.rs#L295-L383), or libstd (https://github.com/rust-lang/rust/blob/master/library/std/src/sys/unix/mod.rs#L240-L289), or libunwind (https://github.com/rust-lang/rust/blob/master/library/unwind/src/lib.rs#L39-L72) rather than a target spec.

Reply:
ok, I'll remove these here and create PRs for the mentioned crates